api/raml/core-command.raml

```diff
@@ -38,9 +38,11 @@ schemas:
         responses: 
             "200": 
                 description: String as returned by the device/sensor via the device service.
+            "400":
+                description: if the request is malformed or unparsable
             "404": 
                 description: if no device exists by the id provided
-            "423": 
+            "422": 

```

Comment:
@brandonforster We were talking about this in stand-up. Probably want to keep 423 here.